app/assets/stylesheets/css/bootstrap-extended.css

```diff
@@ -490,8 +490,8 @@ Bootstrap Enhance */
 .dropdown-toggle:focus {
   outline: 0; }
 
-.navbar-horizontal .dropdown-toggle::after {
-  top: -9px !important; }
+/*.navbar-horizontal .dropdown-toggle::after {*/
```

Comment:
how this change affected admin layout? did you test it?

Reply:
this style made arrow not be alined

Reply:
application layout header is not affected with that change
![screenshot from 2018-01-09 13-13-59](https://user-images.githubusercontent.com/23340152/34718447-5648d7de-f53f-11e7-99a6-adb4c4432c09.png)

Reply:
why you keep this comment in CSS? do you need it? IF NOT - remove